analysis/classes/ui.py

```diff
@@ -87,12 +93,12 @@ def __init__(self, data_blob, result, cfg, predictor_cfg={}, deghosting=False):
         self.num_images = len(data_blob['input_data'])
         self.index = self.data_blob['index']
 
-        concat_result = cfg['trainval']['concat_result']
-        check_concat = set(self.CONCAT_RESULT)
-        for i, key in enumerate(concat_result):
-            if not key in check_concat:
-                raise ValueError("Output key <{}> should be listed under "\
-                    "trainval.concat_result!".format(key))
+        # concat_result = cfg['trainval']['concat_result']
```

Comment:
I disagree with this change. This means we have a default value for `concat_result` in two different places effectively. The `process_config` function already sets `cfg['trainval']['concat_result']` properly (to the default list of strings if necessary). Why does the analysis tools need to repeat the work of `process_config` ?

Reply:
Doesn't this change accomplish what you propose? (i.e., removing checking `concat_result` in two different places?) I'm seeing the code block `if not key in ...` being removed?

Reply:
Ah yes, my bad. In that case we might as well remove the attribute `CONCAT_RESULT` to avoid confusion?